cm_predict.py

```diff
@@ -203,65 +203,55 @@ def predict(self):
         return
 
     def mosaic(self):
-        """
-        Make a mosaic output from obtained predictions
-        Next step: Take into account overlapping argument
-        """
-
+        # Creates /prediction/<product_name> directory
         big_image_product = os.path.join(self.big_image_folder, self.product_name)
         if not os.path.exists(big_image_product):
             os.mkdir(big_image_product)
 
         # Create list of prediction images
         image_list = []
-
         for subfolder in os.listdir(self.prediction_product_path):
             if os.path.isdir(os.path.join(self.prediction_product_path, subfolder)):
                 image_list.append(pathlib.Path(os.path.join(self.prediction_product_path, subfolder, "prediction.png")))
 
-        # Sort images by asc
+        # Sort images by asc (e.g. 0_0, 0_1, 0_2)
         image_list.sort(key=lambda var: get_img_entry_id(var))
 
-        # Rotate each image in the list 270˚ counter clockwise
-        rotateImages(270, image_list)
-
-        # Raster mosaic
         """
         A function that creates raster mosaic.
         As parameters it takes: list of images, number of tiles per row and number of columns
         
         1) Takes the sub-tile width and height from the first image in the list
         2) Sets final image size from col*width, rows*height
-        3) Creates final image from all sub-tiles, and bounding box parameters are also set 
+        3) Creates final image from all sub-tiles, bounding box parameters are also set 
         """
-        overlap_pix = self.overlapping*self.tile_size
+        overlap_pix = self.overlapping * self.tile_size
         if (overlap_pix % 2) != 0:
-            raise Exception ('Even number of pixels needed')
+            raise Exception('Even number of pixels needed')
 
-        crop_coef = int(overlap_pix/2)
+        crop_coef = int(overlap_pix / 2)
```

Comment:
I'm almost sure that
y = x // 2
should work the same as y = int(x / 2) in python 3

Reply:
Indeed

Reply:
But why we need to change it if it's working anyway?

Reply:
Regarding white spaces, it is IDE that formatted, so I just left it as it has formatted.

y = x // 2 is smaller and neat, if to compare with y = int(x / 2)

Reply:
UPD: Python rounds the result to the whole number, but since overlap_pix is float, crop_coef becomes also float which not acceptable